crates/polars-io/src/line/options.rs

```diff
@@ -0,0 +1,150 @@
+use std::path::PathBuf;
+
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "dsl-schema", derive(schemars::JsonSchema))]
+pub struct LineReadOptions {
```

Comment:
Please only add the options listed in the issue. That is, `n_lines`, `encoding`, `eol_char`.